pkg/systemd/overview-cards/tuned-dialog.jsx

```diff
@@ -303,14 +300,14 @@ const TunedDialog = ({
                className="ct-m-stretch-body"
                isOpen
                help={help}
-               onClose={Dialogs.close}
+               onClose={() => dialogResult.resolve()}
```

Comment:
The equivalent would be dialogResult.resolve

Reply:
Not necessarily. The `this` inside the call is different, and that might matter.